### DISCUSSION TITLE
Unable to access free plan of Copilot for students after receiving renewal email

### DISCUSSION THREAD

**Discussion by iamsabbirsobhani** [community/community#44062] [Copilot Conversations]
![image](https://user-images.githubusercontent.com/69682045/211972843-53639b91-39cd-4ff8-9c36-9b85a9099d47.png)

I was getting this email on 9-Jan-2023, and was able to use copilot normally until the 11th. I have been renewed until 08-Feb-2023, but I am unable to use copilot from today 12-Jan-2023, even though the renewal has not expired!

**Comment by aurates** [comment]
same

**Comment by ilaomianyang** [comment]
same

**Comment by Linloir** [comment]
same issue.

**Comment by s-alad** [comment]
same issue.

**Comment by Kogo0813** [comment]
Same

**Comment by aurates** [comment]
Looks like we can use them again right now

**Comment by iamcjwilliams** [comment]
Thank you for taking the time to report and we apologize for the temporary technical issue on our end that may have caused authentication failures. If you are a verified student please retry logging into your IDE plugin, and Copilot should work as expected.

If you run into any further issues, please reply back in detail with any useful screenshots
